Fix release

anbraten opened the pull request:
- Fixes https://ci.woodpecker-ci.org/repos/8958/pipeline/256/5

`cd` was sending the user to the home-directory `/root`, but not to the woodpecker workspace `/woodpecker/src` using `cd ..` solved the problem

- removed deprecated `--charts-repo` arg from `cr`

pat-s commented:
I'd vote to change the branch name in `woodpecker.github.io` to `main` instead of adding a workaround here via `--pages-branch master`?

anbraten commented:
> I'd vote to change the branch name in `woodpecker.github.io` to `main` instead of adding a workaround here via `--pages-branch master`?

We would first need to change that in some other places as well. Would do that in a new PR as this setting was not touched in here. Just fixed the `cd` vs `cd ..` and removed `--charts-repo` option in here. The rest is just replacing `&&` with separate step command lines.

pat-s reviewed (approved):
Alright, let's move forward here then :)
